tests/components/squeezebox/test_sensor.py

```diff
@@ -19,7 +20,7 @@ async def test_sensor(hass: HomeAssistant) -> None:
         ),
         patch(
             "homeassistant.components.squeezebox.Server.async_query",
-            return_value=FAKE_QUERY_RESPONSE,
+            return_value=copy.deepcopy(FAKE_QUERY_RESPONSE),
```

Comment:
@pssc I just update the tests PR, should I change this there? Why do we need a copy rather than just using the constant?

Reply:
the data structure is modified in place from the status call, to ensure its all good for the HA platforms and consistently present.